api/apps/articles/model.coffee

```diff
@@ -165,17 +166,28 @@ toQuery = (input, callback) ->
     query.section_ids = ObjectId input.section_id if input.section_id
     query.biography_for_artist_id = ObjectId input.biography_for_artist_id if input.biography_for_artist_id
     query.tags = { $in: input.tags } if input.tags
-    query.$or = [
+
+    # Only add the $or array for queries that require it (blank $or array causes problems)
+    query.$or ?= [] if input.super_article_for or input.artist_id or input.all_by_author
+
+    # Convert query for articles by author
+    query.$or.push(
       { author_id: ObjectId(input.all_by_author) }
       { contributing_authors: { $elemMatch: { id: ObjectId input.all_by_author} } }
-    ] if input.all_by_author
+    ) if input.all_by_author
 
     # Convert query for articles featured to an artist or artwork
-    query.$or = [
+    query.$or.push(
       { primary_featured_artist_ids: ObjectId(input.artist_id) }
       { featured_artist_ids: ObjectId(input.artist_id) }
       { biography_for_artist_id: ObjectId(input.artist_id) }
-    ] if input.artist_id
+    ) if input.artist_id
+
+    # Convert query for super article for article
+    query.$or.push(
```

Comment:
:+1: works for me—but now that you're just doing one condition I don't think you need to loop this in to $or, you can build on the query similar to how `biography_for_artist_id` does it.

`query['super_article.related_articles']= ObjectId(input.super_article_for) if input.super_article_for`